determined-ai/determined
# fix: remove `noImplicitAny`

## Description
WEB-575



## Test Plan
- Check if all types are correct
- Check if everything works as expected
- Check if test cases pass



## Commentary (optional)
- I avoid using `any` as much as possible, but I had to use `any` in some cases. 



## Checklist
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
